### DISCUSSION TITLE
Train in one language and synthesize in English

### DISCUSSION THREAD

**Discussion by lucasgris** [yl4579/StyleTTS2#211] [Q&A]
Is it possible to train using one language and use the model to synthesize English? I have a dataset in my language, but when I trained the model, it outputs with an undesirable accent.

**Comment by rlenain** [comment]
Hi -- yes, you can. Typically when I train StyleTTS2 to synthesise another language, I keep a bit of English in the dataset, just to make sure I retain the English speaking capability.

The other thing is that the reference you use often matters. If you are using a reference from a (for example) Spanish speaker and try to synthesise English, then you might run into an issue where you get a Spanish accent.

If none of this works, you can try to use this multilingual PL-BERT model on Huggingface - I just open-sourced it. https://huggingface.co/papercup-ai/multilingual-pl-bert

**Reply by ara-vardanyan** [reply]
Hi! I have a use case that requires a synthesis with multiple languages (such as Spanish and English in the same sentence). Do you have any advice on using multilingual PL-BERT to fine tune for this use case?

**Comment by lucasgris** [comment]
Got it, thanks for your reply! I was imagining that mixing the languages might be a solution.

I didn't pay attention to the reference, though. I have this problem that I don't have the reference in English, so maybe I'll think how can I get to work in this case. Do you have any ideas?

Also, good work with the PL Bert. It will definitely help me and others!

**Comment by rlenain** [comment]
You can maybe use RVC (or any voice conversion model) to convert an english reference to the target speaker you are trying to synthesise, and then use the RVC output as the reference for styletts2

**Comment by ADD-eNavarro** [comment]
May I suggest using [Hexgrad](https://hexgrad.com/#UbDoPLxo7L1wOQU7Z8eHvTjYRbsCTSM-bHjdvbx9Oj38FC09VgYEPjwPrj1N3PU9EMsSPTbNpL1IGbW8PRMaPlQTrj1Y19C7iP3dPdm7jD2LsLE9FneHvYRm-rsOa9G9Uq1NvZWsGb4aevw9APJFvtapZj2w_ye-hMAkvNBfaD0jo8s9YkslPbzBCr4Mbi0-DM0LPtx4pbxcrxq8CvoBPdy9STuJObK9dM8bvg79vb1VYxu-uT_2vpmcBT2zDwo-F11Fvclxtrt85Nw8j3VpPdUca73Oko49wBjxvLMbhL4C2-w9TGH8O5z1C71unMG9yjQgP6CgHbwX5Ig9vkQNPQgjeLxhpkY9eEEGPhKHFD12kj89LzwWvsU35jyXjzg91YZpPXvC6LtOywq98GwAviFjc73-iw49o6-zvZwtPD1WhUa96v1mvM7_e73iuA092vwiPpfsIj5S6t08-CBLPVw8sz0KGwA-40eXvraSx71e5kC-hcO9vVmdLT5GD1O9Tk0dPdT6EL7U5tS9cMcQvT9A2bsyQlM-2sLlPBiVbD0uJhq99mwUPh6jVj7KE809zBUYvnUFNb4ank4-VBkGvaKKPT7AJNa8aB7kPCZ2pTztNoG7pJM1PNeRUT2QcV28yEOmvPY1_L0aliK91yuZPhQUUD4Mw2u9t2QVPdw7t7vUJT27ayRWPaPCvT6eOS4-y4qKvQ4XTj6jjJ2-2OqpvTcNUL6utIU9VGopPKbcwj59j4S82e85vpC0gj1mLdY-aLU8vqRorLx8jA0_StWgPtxj-rtoiG--zPi_vuL0aT3cMIy9Wv5kPAW5KD1OTsW-CFO9PBYqy71eIrm-gcU-PidAgz08AZO8RFR1vrcGUj6g0oy7B1itvm6h87wBS609c2vGvvSG8Tyeyie_Tv3MvgzOb75XdaI-qiYxPnhVdL4NbQO_aECevaQQYD6djMO8SaWZPLjMDz_hdjO-u80GvtEl9T5dKYy92yysPr-TCT7zc2E-OHOBvnr6871-O8g9tdh3PXB4ob6Wzs4-uIxgvkA0Pb1GUHM-Hq8dPnBajj4Hy10-ou-fPrDJ0L13ZAi_bOR-vvi6iz6DRLG-Vi6wvvScrzwx31G-ShCYvqFfTr54vqs-rnKaPGf_SD68_pw-Bi7mvaZzKj44tNi9qossPgHErb5V-yq-G4WBvficzLyKDQK-DEQEvse8zr06-6g9SEkvPmmk8j7YH409btoNvG9jxL2wQSK-jl-JvhaXW71jOF296oOfvlQ2wr0I4g098CE1Pv341736XVM9vuArPVhS5r1tWbs-tjKJPkGZuD6VEU69mf2jPYB_qz2Mn709ZNFbPsJU8D5FrQs-KuQTvg)? You can send in a reference audio in any language, with English text, and since it's the standard (English only) model under the tool, you will get non-accent English audio.
